Formula/ThorsSerializer.rb

```diff
@@ -0,0 +1,59 @@
+class Thorsserializer < Formula
+  desc "Declarative Serialization Library (Json/Yaml) for C++"
+  homepage "https://github.com/Loki-Astari/ThorsSerializer"
+  url "https://github.com/Loki-Astari/ThorsSerializer.git",
+    :tag => "1.5.7",
```

Comment:
: under "

Reply:
Sorry. I am not sure what you mean.

Reply:
more spaces until the : in :tag is under " in "https://github.com/Loki-Astari/ThorsSerializer.git",

Reply:
> : under "

Didn't know I'd managed to win this indentation battle! 🎉

Reply:
No problem.

Reply:
@DomT4 until the Ruby police decide it should be a two-space continuation because they said so.

Reply:
@ilovezfs I'll go along with a two-space indent on most things but aligning URLs like that bothers me more than it should, heh.